tensorflow_serving/util/json_tensor.cc

```diff
@@ -637,14 +638,28 @@ Status FillPredictRequestFromJson(
   auto itr_instances = doc.FindMember(kPredictRequestInstancesKey);
   auto itr_inputs = doc.FindMember(kPredictRequestInputsKey);
   if (itr_instances != doc.MemberEnd()) {
-    if (itr_inputs != doc.MemberEnd()) return FormatError(doc);
-    if (!itr_instances->value.IsArray()) return FormatError(doc);
-    if (!itr_instances->value.Capacity()) return FormatError(doc);
+    if (itr_inputs != doc.MemberEnd()) {
+      return FormatError(doc, "Expecting only one of '",
+        string(kPredictRequestInputsKey), "' or '",
```

Comment:
any reason why you need to wrap kXXX in string()? plain kXXX const should work. don't they?

Reply:
It's not needed, removed. Artifact of when I was using the plus operator which needed the casting.